src/pages/HomePage.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import FetchStats from '../store/api';
+import { GetStats } from '../store/reducer';
+
+const HomePage = () => {
+  const CountryStore = useSelector((store) => store.details);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (CountryStore.length === 0) {
+      FetchStats()
+        .then((response) => dispatch(GetStats(response)));
+    }
+  }, []);
+
+  const Africa = CountryStore.filter((item) => item.continent === 'Africa');
+
+  return (
+    <div className="homePage">
+      <h1 className="continent">Africa</h1>
+      <ul className="dataUL">
+        {
+        Africa.map((country) => (
+          <Link key={country.country} to={{ pathname: `/country/${country.country}` }}>
```

Comment:
- [ ] In your project, it's not possible to navigate to the detail page by clicking the individual category item, in the home page, please fix this, see video below 👍🏿 

https://user-images.githubusercontent.com/69638013/180864367-9d73ef52-6198-41e4-bef3-cf1e48e0da59.mp4